languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt

```diff
@@ -138,6 +138,9 @@ class ServerRenderer @Inject constructor(
                         |        params: {
                         |          <${it.resource().fullUri.variables.map { "$it: requiredString" }.joinToString(separator = ",\n")}>
                         |        },
+                        |        query: {
```

Comment:
This adds a query validation option (see https://hapi.dev/api/?v=20.0.2#-routeoptionsvalidatequery), which uses the generated joi validation schema to specify the format of the query parameter.